storage/files/files.go

```diff
@@ -56,7 +56,7 @@ func (s Storage) Save(page *storage.Page) (err error) {
 func (s Storage) PickRandom(userName string) (page *storage.Page, err error) {
 	defer func() { err = e.Wrap("can't pick random page", err) }()
 
-	path := filepath.Join(s.basePath, page.UserName)
+	path := filepath.Join(s.basePath, userName)
 
 	files, err := os.ReadDir(path)
```

Comment:
do check for the presence of the user's folder, if it is not there, send a corresponding message